src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      name="bigSearchForm"
+      data-qa="big"
+    >
+      <label>
+        <img
+          src="/images/Search.svg"
```

Comment:
It is recommended to use a more descriptive alternative text for the search icon. Replace 'searchLoup' with 'Search icon' in the alt attribute.